torch/ao/quantization/backend_config/executorch.py

```diff
@@ -47,6 +53,42 @@
     is_dynamic=True,
 )
 
+# xnnpack compatible dtype configs
+
+# We restrict scale values to be 2 ** -12 to ensure the
+# requantization scale never falls below the xnnpack lower
+# threshold. Additionally, for qint8 weight, we restrict
+# the quantization values to [-127, +127], excluding -128.
+# For more detail, refer to the description of
+# `default_symmetric_qnnpack_qconfig`.
+
+# TODO: add additional restriction on qscheme to ensure it
+# is either per_tensor_symmetric or per_channel_symmetric
+
+executorch_act_qint8_scale_min_2_neg_12 = DTypeWithConstraints(
```

Comment:
is executorch a user facing concept in OSS?  If not, is it ok to use the name here?

Reply:
looks like it's not oss'ed yet, this file was added at the time that we didn't give much thought about this. this will be oss'ed and we probably have mentioned executorch in the recent pt conference so I feel it should be OK?